test/sass/_value.scss

```diff
@@ -114,5 +114,16 @@
       'Gets a value from a nested map');
   }
 
+  @include test('Map Safe Get [function]') {
+    $map: (
+      one: 'two',
+      ),
+    );
+    $expect: 'two';
+    
+    @include assert-equal(map-safe-get($map, one), $expect,
+      'Safely return a value from a map');
+  }
```

Comment:
I would like to also test error cases, but unfortunately we cannot handle `@error` in sass or with sass-true.